CHANGELOG.md

```diff
@@ -379,13 +419,13 @@
             a = []
             for _ in range(length):
                 a.append( num & ((1 << num_bits_shift) - 1) )
-                num = num >> num_bits_shift
+                num = num >> num_bits_shift 
             return tuple(a)
 
         def pack(z, num_bits_shift: int) -> int:
             limbs = (z.d0, z.d1, z.d2)
             return sum(limb << (num_bits_shift * i) for i, limb in enumerate(limbs))
-
+            
```

Comment:
```suggestion

```

Reply:
The hint implemented in this entry includes the whitespace so I think it's clearer if we also add it on the changelog. Although, now that I think about it a bit more, maybe we should add a comment to clarify why there are two identical hints. Something like: _"Note: this hint is similar to the one in #983, but with some trailing whitespace removed"_
What do you think?

Reply:
Sorry, I missed that this was hint code 🤦